frontend/packages/local-storage-operator-plugin/integration-tests/views/disks-list-page.views.ts

```diff
@@ -0,0 +1,30 @@
+import { $, $$ } from 'protractor';
+import { resourceRows } from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+
+export const page = {
+  diskTab: $('a[data-test-id="horizontal-link-Disks"]'),
+  resourceTitles: $$('[data-test-id="resource-title"]'),
+};
+
+export const diskList = {
+  headers: ['Name', 'Disk State', 'Type', 'Model', 'Capacity', 'Filesystem'],
+  dropdownFilter: $('button[data-test-id="filter-dropdown-toggle"]'),
+  dropdownFilterHeader: $('.pf-c-dropdown__group-title'),
+  textFilter: $('input[data-test-id="item-filter"]'),
+};
+
+export const clickFilterDropdown = () => diskList.dropdownFilter.click();
+
+export const clearDiskFilter = async () => {
+  const uncheckFilter = await $('.pf-c-chip-group__close >  button');
+  await click(uncheckFilter);
+};
+export const checkDiskFilter = async (filter: string) => {
+  await clickFilterDropdown();
+  const countBadge = $(`a[data-test-row-filter="${filter}"] > div > .pf-c-badge`);
+  const count = await countBadge.getText();
+  await click($(`#${filter}`));
+  const rowsCount = await resourceRows.count();
+  expect(rowsCount).toBe(Number(count));
+};
```

Comment:
These functions can go inside the diskList object as well.

Reply:
https://github.com/openshift/console/pull/6606#discussion_r490740481